src/payment/strategies/googlepay/googlepay-adyenv3-initializer.ts

```diff
@@ -0,0 +1,110 @@
+import { round } from 'lodash';
+
+import { Checkout } from '../../../checkout';
+import PaymentMethod from '../../payment-method';
+
+import { BillingAddressFormat, GooglePaymentData, GooglePayInitializer, GooglePayPaymentDataRequestV2, TokenizePayload } from './googlepay';
+
+export default class GooglePayAdyenV3Initializer implements GooglePayInitializer {
```

Comment:
Didn't find a place, where do you use this class. It only used in commented code above. Do you need this one?